dts/arm/renesas/rcar/gen4/rcar_gen4_cr52.dtsi

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2022 IoT.bzh
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <arm64/armv8-r.dtsi>
+#include <zephyr/dt-bindings/interrupt-controller/arm-gic.h>
+#include <zephyr/dt-bindings/clock/renesas_cpg_mssr.h>
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "arm,cortex-r52";
+			reg = <0>;
+		};
+	};
+
+	timer {
+		compatible = "arm,armv8-timer";
+		interrupt-parent = <&gic>;
+		interrupts = <GIC_PPI 13 IRQ_TYPE_LEVEL
+				 IRQ_DEFAULT_PRIORITY>,
+				<GIC_PPI 14 IRQ_TYPE_LEVEL
+				 IRQ_DEFAULT_PRIORITY>,
+				<GIC_PPI 11 IRQ_TYPE_LEVEL
+				 IRQ_DEFAULT_PRIORITY>,
+				<GIC_PPI 10 IRQ_TYPE_LEVEL
+				 IRQ_DEFAULT_PRIORITY>;
+	};
+
+	soc {
+		interrupt-parent = <&gic>;
+
+		sram0: memory@40040000 {
+			compatible = "mmio-sram";
+			reg = <0x40040000 0x1fc0000>;
+		};
+
+		gic: interrupt-controller@f0000000 {
+			compatible = "arm,gic";
+			reg = <0xf0000000 0x1000>,
+				  <0xf0100000 0x20000>;
+			interrupt-controller;
+			#interrupt-cells = <4>;
+			status = "okay";
+		};
+
+		/* Clock controller
+		 * Using domain 0 as linux
+		 * Should use domain 3 later
+		 */
+		cpg: clock-controller@e6150000 {
+			compatible = "renesas,rcar-cpg-mssr";
+			reg = <0xe6150000 0x4000>;
```

Comment:
`0x4000` is same as linux ? Looks to end at `0x3c00` in docs (without clock monitoring)

Reply:
Same as linux : https://github.com/renesas-rcar/linux-bsp/blob/v5.10.147/rcar-5.2.0.rc6/arch/arm64/boot/dts/renesas/r8a779f0.dtsi#L368